goodbye-graffiti-agent/maps-function/package.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Download and run the `addlicense` CLI tool in this directory to insert the correct license headers and make the license-check test pass: https://github.com/google/addlicense

Reply:
Thanks Kris, I was unable to run the script. For now I've just added the correct license header. We can troubleshoot this later